sdv/multi_table/base.py

```diff
@@ -56,12 +57,17 @@ def _initialize_models(self):
                 **synthesizer_parameters
             )
 
-    def __init__(self, metadata, locales=None):
+    def __init__(self, metadata, locales=None, synthesizer_kwargs=None):
```

Comment:
I think the `synthesizer_kwargs` method isn't meant to be part of the new API and we can actually just drop it in the base and HMA
@npatki Is that correct?

Reply:
@amontanez24 correct!